setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup
+
+setup(
+    name="CodaV2PythonClient",
+    version="0.1.0",
```

Comment:
I would start with 0.0.0 until we have a released version

Reply:
Thanks Updated.